### PR TITLE
scx_bpfland: Documentation consistency fix

### PR DESCRIPTION
**Pull request by anh0516**

### CODE REVIEW

**Comment by arighi** [comment]
Looks good. I see the PR contains some merges from main, probably they're dropped if we merge, but just to make sure, can you rebase and include only the last commit? Thanks!

**Comment by anh0516** [comment]
I reverted my branch so it only contains the one commit. Is there a way for me to pull in changes since then without creating extraneous commits? I can't find one. Easy solution is to just wait until this is merged.

**Review by arighi — APPROVED** [review]
Looks good now, thanks.

**Pull request merged**
